be/ut/extmap.c

```diff
@@ -109,23 +109,25 @@ static void test_obj_fini(struct m0_be_tx *tx)
 
 static void test_init(void)
 {
-	struct m0_be_domain_cfg cfg;
+	struct m0_be_domain_cfg *cfg;
 	struct m0_be_tx_credit	cred = {};
```

Comment:
If we allocate the "cfg" from heap, the stack is enough to hold "cred"?
No need to move "cred" to heap?

Reply:
Yes.